src/NewPerson/NewPerson.jsx

```diff
@@ -0,0 +1,211 @@
+import React, { useState, useEffect } from 'react';
+import { useHistory, useRouteMatch } from 'react-router-dom';
+import { PropTypes } from 'prop-types';
+import classNames from 'classnames';
+import { validName, getPeople, initErrors,
+  initPerson } from '../helpers/Helpers';
+
+export const NewPerson = ({ people, setPeople }) => {
+  const [personDate, setPersonDate] = useState(initPerson);
+  const [peopleDate, setPeopleDate] = useState();
```

Comment:
date - дата
data - данные
```suggestion
  const [peopleData, setPeopleData] = useState();
```
Но вообще думаю тут можно обойтись без слова `data`. просто `people` и `setPeople`